tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "module": "es6",
+    "target": "es5",
```

Comment:
[no action] 👍🏻   preserving ES5 compatibility. It's so weird to me to think that ES6 is mostly just assumed now 👴🏻